Makefile

```diff
@@ -389,9 +389,12 @@ install/yaml/: TAG = $(BASE_VERSION)
 endif
 install/yaml/: update-chart-deps install/yaml/install.yaml install/yaml/01-open-match-core.yaml install/yaml/02-open-match-demo.yaml install/yaml/03-prometheus-chart.yaml install/yaml/04-grafana-chart.yaml install/yaml/05-jaeger-chart.yaml install/yaml/06-open-match-override-configmap.yaml install/yaml/07-open-match-default-evaluator.yaml
 
+# We have to hard-code the Jaeger endpoints as we are excluding Jaeger, so Helm cannot determine the endpoints from the Jaeger subchart
```

Comment:
So... I get why you're doing this, but it's really uggly.  Are there any other setups which would mean this doesn't have to happen?  Just probing for options.

Reply:
Not easily. We're generating a config that includes Jaeger settings, separately from deploying Jaeger itself. To fix this, we'd want to be able to have the jaeger-deployment step generate someting that Open Match could pick up to gain knowledge of where Jaeger lives.

We use Jaeger Operator with side-car injection in our project, which removes all of this pain.

Reply:
I just realised I wrote "so Helm" instead of "as Helm" in this comment. And left out that the reason for this is because we're not installing the relevant subchart at the time.

When using Helm to install Helm (rather than trying to generate partial manifests), this isn't an issue, the Helm templates just ask the sub-charts's named template for the relevant name.

Reply:
The current stacking of the config map was an attempt to simplify things, specifically for the yaml install path.  The blame goes on me for suggesting it, but I think it's proving to be less helpful than I thought.